### PR TITLE
[jsk_perception] add check-detection-srv

### PR DESCRIPTION
**Pull request by h-kamada**
* catch detection result from detection server in fridge demo

### CODE REVIEW

File: jsk_perception/euslisp/detection_interface_servicecall.l [hunk 1]
```diff
@@ -1,11 +1,12 @@
-(defun check-detection (&key (timeout 10) ((:type atype) nil) (tf-force nil)
-                             (single nil) (speak t)
-                             (speak-name)
-                             (detection-topic *detection-topic*)
-                             ((:target-object target-obj) (make-cube 60 60 60)))
+(defun check-detection-srv (&key (timeout 10) ((:type atype) nil)
```
Comment (k-okada) [inline jsk_perception/euslisp/detection_interface_servicecall.l]: is this ok to change function name? do we need to update code which uses this function? @h-kamada

**Comment by h-kamada** [comment]
sorry for late responce.
I changed name to avoid updating all names currently.
We have to launch server node to use this func, and I thought some people may think unwelcome.

**Comment by k-okada** [comment]
6/21 昼の議論：topicでよい？

**Comment by k-okada** [comment]
@h-kamada これ，今もつかっている？マージしたほうが良い？

**Comment by h-kamada** [comment]
topicでも問題無さそうという結論で変わらなそうなので、closeしときます。

**Pull request closed without merging**